project_stock/models/project_task.py

```diff
@@ -123,6 +123,14 @@ def _compute_unreserve_visible(self):
             any_quantity_done = any([m.quantity_done > 0 for m in item.move_ids])
             item.unreserve_visible = not any_quantity_done and already_reserved
 
+    def _set_group_id(self):
```

Comment:
Remember that you are adding this in a general model, so the naming of the methods should be self-explanatory as possible:

```suggestion
    def _set_procurement_group_id(self):
```